container-disc/src/test/java/com/yahoo/container/jdisc/component/DeconstructorTest.java

```diff
@@ -51,6 +50,18 @@ public void require_shared_resource_released() {
         assertTrue(sharedResource.released);
     }
 
+    @Test
+    public void bundles_are_uninstalled() throws InterruptedException {
+        var bundle = new UninstallableMockBundle();
+        // Done by executor, so it takes some time even with a 0 delay.
+        deconstructor.deconstruct(emptyList(), singleton(bundle));
+        int cnt = 0;
+        while (! bundle.uninstalled && (cnt++ < 12000)) {
+            Thread.sleep(10);
```

Comment:
Consider reducing sleep to `Thread.sleep(1)` to minimize runtime of test.

Reply:
This is millis, so 1/100 of a sec. Is that too much?

Reply:
Done

Reply:
oh.. too late